sites/geohub/src/components/DataView.svelte

```diff
@@ -330,19 +330,18 @@
     <div
       hidden={!$indicatorProgress}
       class="loader-container">
-      <Loader />
+      <Loader
+        class="m-auto"
+        size="small" />
```

Comment:
According to the screenshot you pasted, I feel new loader is a bit smaller for data tab. Personally, original size of loader is much better.